middlewares/contentTypeCheck.js

```diff
@@ -7,8 +7,12 @@
  * @param {function} next - Express middleware function
  */
 module.exports = (req, res, next) => {
-  if (req.headers['content-type'] && req.headers['content-type'] !== 'application/json') {
-    return res.boom.unsupportedMediaType(`Invalid content-type header: ${req.headers['content-type']}, expected: application/json`)
+  if (req.headers['content-type'] && (req.headers['content-type'] !== 'application/json')) {
+    const contentType = req.headers['content-type']
+    const notMultiPart = !(contentType.includes('multipart/form-data'))
+    if (notMultiPart) {
+      return res.boom.unsupportedMediaType(`Invalid content-type header: ${req.headers['content-type']}, expected: application/json or multipart/form-data`)
+    }
```

Comment:
Nit: Since we are using `req.headers['content-type']`, why not define the `contentType` before the if statement and use it everywhere?